test/FixtureFactory.php

```diff
@@ -61,4 +66,9 @@ public static function createUncapturedCharge()
 
         return Charge::fromArray(self::createPorter()->importOne(new ImportSpecification($createCharge)));
     }
+
+    private static function createCardFromNumber($cardNumber, $cardCvc = '123')
```

Comment:
* This method should just be called *createCard* since we [should not describe method parameters](http://codelegance.com/semantic-method-naming/#donotdescribeparameters).
* There doesn't seem to be any compelling reason to promote CVC to a parameter since its value is constant.